.changeset/lemon-shoes-hug.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
This doesn't seem right 🤔 

How can you ship a new version of `@vercel/fs-detectors` without a patch?

Reply:
Nvm. Seems that `LocalFileSystemDetector` had a bug, so this will be a semver major change.